dump/mapping/__init__.py

```diff
@@ -0,0 +1 @@
+import builders
```

Comment:
I have never before seen this "circle" icon:
![image](https://github.com/user-attachments/assets/e97f486e-c64c-4ac1-8aec-628b5285ef31)
I don't know what it means, unfortunately.